scripts/test.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# coding:utf-8
+
+import os
+import unittest
+
+from setup import Android, Project, Ios, Flutter
+
+project = Project()
+project.project_path = os.path.curdir
+project.new_package = 'com.example'
+project.new_app_name = ''
+project.new_project_name = 'example_project'
+
+
+class AndroidTest(unittest.TestCase):
+    def setUp(self):
+        self.android = Android(project)
+
+    def test_get_old_package(self):
+        self.assertEqual(self.android.get_old_package(), 'co.nimblehq.flutter.template')
+
+    def test_get_old_app_name(self):
+        self.assertEqual(self.android.get_old_app_name(), 'Flutter Templates')
```

Comment:
@manh-t as you are using `make test`, this `Flutter Templates` or the template package id can be accessed via env.PACKAGE_NAME or env. APP_NAME right?

Reply:
@sleepylee I think it's possible, let me try 🦖